Bottom: 0efecf947edbbd6da7475ec629e919f5e60f5ee6
Top:    86ed9f8eb07f0149429219c4be2e49a397e16fe8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-15 22:19:07 +0000

Refresh of tmp-changes

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 0963c21..e9f2efc 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1703,8 +1703,8 @@ Collect (char poly, jmp_buf * e, PLINE * a, POLYAREA ** contours, PLINE ** holes
   VNODE *cur;
   DIRECTION dir = UNINITIALISED;
 
-//  cur = (&a->head);
-  cur = (&a->head)->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
+  cur = (&a->head);
+//  cur = (&a->head)->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
   do
     {
 #if 0
